### PR TITLE
README.md: Add responsible disclosure instructions.

### PR DESCRIPTION
**Pull request by ZmnSCPxj-jr**
Closes: #39

### CODE REVIEW

**Comment by SeverinAlexB** [comment]
I compared the attached keys with the keys posted in the Telegram group and they match.
I can't find any issues with this PR. Merge ok IMO.

@yaslama @tnull Maybe you wanna hunch over and give is your go/no-go?

File: README.md [hunk 1]
```diff
@@ -50,6 +50,59 @@ List of Lightning Service Providers in alphabetic order that currently or will s
...
+## Responsible Disclosure
 
+As [CVE-2019-12998][], [CVE-2019-12999][], and [CVE-2019-13000][] show, it
+is possible to have CVE-level bugs in the specification.
 
+[CVE-2019-12998]: https://nvd.nist.gov/vuln/detail/CVE-2019-12998
+[CVE-2019-12999]: https://nvd.nist.gov/vuln/detail/CVE-2019-12999
+[CVE-2019-13000]: https://nvd.nist.gov/vuln/detail/CVE-2019-13000
 
+Thus, the LSPS participants have also given the following contact persons
```
[the remark this answers is not in the document]
Comment (ZmnSCPxj-jr) [reply README.md]: Done, except I used just "LSPS" because "LSPS specification" is like "ATM machine".

**Comment by ZmnSCPxj-jr** [comment]
Updated: reword requests to report security-crticial issues.

**Comment by ZmnSCPxj-jr** [comment]
Ready to merge?

**Comment by yaslama** [comment]
Ok for me

**Pull request merged**